README.md

```diff
@@ -416,7 +287,7 @@ We are grateful to the community for contributing bugfixes and improvements! Ple
 
 ## License
 
-Apache 2 Licensed. See [LICENSE](https://github.com/terraform-aws-modules/terraform-aws-rds-aurora/tree/master/LICENSE) for full details.
```

Comment:
Looks like I messed this up over 3 years ago 😅  https://github.com/terraform-aws-modules/terraform-aws-eks/commit/ee9f0c646a45ca9baa6174a036d1e09bcccb87b1